sdk/textanalytics/ai-text-analytics/src/util.ts

```diff
@@ -84,15 +84,33 @@ export function parseHealthcareEntityIndex(pointer: string): number {
 
 const jsEncodingUnit = "Utf16CodeUnit";
 
-export function addStrEncodingParam<T>(options: T): T & { stringIndexType: StringIndexType } {
-  return { ...options, stringIndexType: jsEncodingUnit };
+/**
+ * Measurement units that can used to calculate the offset and length properties.
+ */
+export type StringIndexType = "TextElements_v8" | "UnicodeCodePoint" | "Utf16CodeUnit";
+
+export function addStrEncodingParam<T>(
+  options: T & { stringIndexType?: StringIndexType }
+): T & { stringIndexType: StringIndexType } {
+  return { ...options, stringIndexType: options.stringIndexType || jsEncodingUnit };
+}
+
+/**
+ * Set the stringIndexType property with default if it does not exist in the options bag.
+ * @param options - operation options bag that has a {@link StringIndexType}
+ * @internal
+ * @hidden
+ */
+export function setStrEncodingParam<T>(
```

Comment:
Do you need all three of these different methods?

Reply:
they were before renaming the property back to the service's name. I removed one.